### PR TITLE
Windows: replace `MSVCRT` with `CRT`

### PR DESCRIPTION
**Pull request by compnerd**
This is part of the migration path to remove the use of the `visualc`
module from the Swift SDK for Windows.  The use of the `stdout` and
`stderr` types requires the use of the Swift overlay as the C library
interfaces are not directly usable with the same name.

### CODE REVIEW

**Comment by compnerd** [comment]
@swift-ci please test

**Comment by compnerd** [comment]
Please do not merge this - this needs to be synchronized with https://github.com/apple/swift/pull/34299

**Comment by abertelrud** [comment]
Sounds good.  Should we mark as Draft to prevent it from getting merged?

**Comment by compnerd** [comment]
@abertelrud - once it is accepted, I think that I should be able to coordinate the changes in short order

**Review by abertelrud — APPROVED** [review]
LGTM

**Comment by abertelrud** [comment]
@compnerd That sounds good — I misunderstood, and was just going off of the note to please not merge yet (which I've seen done in other cases by marking as Draft until it can be merged).  As far as I'm concerned this can be merged any time.  Thanks!

**Pull request merged**
